### PR TITLE
update check to look for `string` 'true' not `bool`

### PR DESCRIPTION
**Pull request by Alan-eMartin**
**Context:**
- PST theme configurations were ignored, causing command palette to appear even though it was configured to be disabled.

**Description of the Change:**
- Use string values of `true` in XST

**Benefits:**
- N/A

**Possible Drawbacks:**
- N/A

**Related GitHub Issues:**
- N/A

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -99,7 +99,7 @@
 
 # Xanadu theme options (see theme.conf for more information).
 html_theme_options = {
-    "command_palette_enabled": False,
+    "command_palette_enabled": True,
```
Comment (Alan-eMartin) [inline doc/conf.py:102]:
```suggestion
    "command_palette_enabled": False,
```

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 
 ### Contributors
 
 This release contains contributions from (in alphabetical order):
 
+- [Alan Martin](https://github.com/alan-emartin)
+
+### Bug fixes
+
+* fixed an issue with the `command_palette_enabled` and `search_on_pennylane` options
```
Comment (Mandrenkov) [inline .github/CHANGELOG.md:11]: # Nitpick
```suggestion
* Fixed an issue with the `command_palette_enabled` and `search_on_pennylane` options
```

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+  having falsey values, big thanks to [Mikhail Andrenkov](https://github.com/Mandrenkov).
+  [(#60)](https://github.com/XanaduAI/xanadu-sphinx-theme/pull/60)
+
 ## Release 0.7.0
 
 ### Contributors
 
-This release contains contributions from (in alphabetical order):
```
Comment (Mandrenkov) [inline .github/CHANGELOG.md:11]: # Question
Why was this line removed?

**Review by Mandrenkov — CHANGES_REQUESTED** [review]
In general, looks good! 👍 

But if we're going to skip the pre-release PR, we should also update the version in `_version.py`.

File: .github/CHANGELOG.md (hunk 3 above)
Comment (Alan-eMartin) [reply .github/CHANGELOG.md:11]: Sorry my mistake.

**Comment by Alan-eMartin** [comment]
> In general, looks good! 👍
> 
> But if we're going to skip the pre-release PR, we should also update the version in `_version.py`.

done! ty!

**Review by Mandrenkov — APPROVED** [review]
